server/database/CouchdbInit.js

```diff
@@ -0,0 +1,44 @@
+var async = require('async');
```

Comment:
Is there a particular reason for choosing to use the async library when the nano library is natively async/can return promises?

Reply:
Other than that, these init helper functions look good! 👍

Reply:
Valid point. I made a note to change it to nano.